src/status/formatter.js

```diff
@@ -1,14 +1,32 @@
-module.exports = (info, userIdToUsername) => {
-  return [...info.values()]
+module.exports = ({ eventStatus, memberStatus }, userIdToUsername) => {
+  
+  return [...memberStatus.values()]
```

Comment:
The logic here is getting complicated, we should break it down into helper functions